moveit_core/planning_request_adapter/src/planning_request_adapter.cpp

```diff
@@ -140,7 +144,9 @@ bool PlanningRequestAdapterChain::adaptAndPlan(const planning_interface::Planner
       fn = [&adapter = *adapters_[i], fn, &added_path_index = added_path_index_each[i]](
                const planning_scene::PlanningSceneConstPtr& scene, const planning_interface::MotionPlanRequest& req,
                planning_interface::MotionPlanResponse& res) {
-        return callAdapter(adapter, fn, scene, req, res, added_path_index);
+        res = callAdapter(adapter, fn, scene, req, added_path_index);
+        // If any adapter fails, return false
```

Comment:
Can you add another sentence here that documents for future code readers why this is necessary? Something like:
```suggestion
        // If any adapter fails, return false. Otherwise, if one of the planning request adapters failed, the pipeline would continue onward anyway and the failure error code would get lost.
```

Reply:
done!